Module Name/manifest.php

```diff
@@ -20,42 +20,94 @@
+$description = 'filler';            // Short text description
 $entryURL    = "index.php";   // The landing page for the unit, used in the main menu
 $type        = "Additional";  // Do not change.
 $category    = '';            // The main menu area to place the module in
-$version     = '';            // Version number
-$author      = '';            // Your name
+$version     = '0.1';            // Version number
+$author      = 'Harry Merrett';            // Your name
 $url         = '';            // Your URL
 
 // Module tables & gibbonSettings entries
-$moduleTables[] = ''; // One array entry for every database table you need to create. Might be nice to preface the table name with the module name, to keep the db neat. 
-$moduleTables[] = ''; // Also can be used to put data into gibbonSettings. Other sql can be run, but resulting data will not be cleaned up on uninstall.
+$moduleTables[] = 'CREATE TABLE `deepLearningEvent` (
+  `deepLearningEventID` int(10) unsigned zerofill NOT NULL AUTO_INCREMENT,
+  `name` varchar(40),
+  `description` text,
+  `backgroundImage` text,
+  `active` boolean,
+  PRIMARY KEY (`deepLearningEventID`)
+)'; // One array entry for every database table you need to create. Might be nice to preface the table name with the module name, to keep the db neat.
+$moduleTables[] = 'CREATE TABLE `deepLearningExperience` (
+  `deepLearningExperienceID` int(10) unsigned zerofill NOT NULL AUTO_INCREMENT,
+  `deepLearningEventID` int(10) unsigned,
+  `deepLearningMajorID1` int(10) unsigned
+  `deepLearningMajorID2` int(10) unsigned
+  `minor1` varchar(30),
+  `minor2` varchar(30),
+  `headerImage` text,
+  `maxEnrollment` int(3),
+  `blurb` text,
+  `timestamp` timestamp,
+  PRIMARY KEY (`deepLearningExperienceID`)
+)'; // Also can be used to put data into gibbonSettings. Other sql can be run, but resulting data will not be cleaned up on uninstall.
+$moduleTables[] = 'CREATE TABLE `deepLearningDate` (
+  `deepLearningEventID` int(10) unsigned,
+  `date` date,
+  `name` varchar(30),
+)';
+$moduleTables[] = 'CREATE TABLE `deepLearningMajor` (
+  `deepLearningMajorID` int(10) unsigned zerofill NOT NULL AUTO_INCREMENT,
+  `name` varchar(30),
+  PRIMARY KEY (`deepLearningMajorID`)
+)';
+$moduleTables[] = 'CREATE TABLE `deepLearningExperienceHost` (
+  `deepLearningExperienceHostID` int(10) unsigned zerofill NOT NULL AUTO_INCREMENT,
+  `gibbonPersonID` int(10) unsigned,
+  `deepLearningExperienceID` int(10) unsigned,
+  PRIMARY KEY (`deepLearningExperienceHostID`)
+)';
+$moduleTables[] = 'CREATE TABLE `deepLearningEnrollment` (
+  `deepLearningEnrollmentID` int(10) unsigned zerofill NOT NULL AUTO_INCREMENT,
+  `deepLearningExperienceID` int(10) unsigned
+  `gibbonPersonID` int(10) unsigned,
+  `enrolDate` date,
+  `choice` enum(`1`,`2`,`3`).
+  `status` enum(`Pending`,`Accepted`)
```

Comment:
Enums should have a default value defined, and this line is also missing a comma. This is where creating the table definitions in the database and exporting them may help ensure they work right.